tests/layer_tests/tensorflow_tests/test_tf_BinaryOps.py

```diff
@@ -123,7 +132,7 @@ def create_add_placeholder_const_net(self, x_shape, y_shape, ir_version, op_type
                               'Equal', 'NotEqual', 'Mod', 'Greater', 'GreaterEqual', 'Less',
                               'LessEqual',
                               'LogicalAnd', 'LogicalOr', 'LogicalXor', 'FloorMod', 'FloorDiv',
-                              'Xdivy'])
+                              'Xdivy', 'BitwiseAnd', 'BitwiseOr', 'BitwiseXor',])
     @pytest.mark.nightly
     @pytest.mark.precommit
```

Comment:
please create a separate test suite for Bitwise because you have added to one which is not included into the pre-commit (no `precommit_tf_fe` mark ).

Reply:
From what I see, 1 case from test data is marked as precommit_tf_fe in inputs list just above first pytest decorator. If this 1 test is not enough, maybe it will be worth to add few more tests for cover all binary ops?